### DISCUSSION TITLE
What would be the best way to handle this

### DISCUSSION THREAD

**Discussion by quarinteen** [requarks/wiki#5800] [Help / Questions]
I have an image that was too big. I made it 25% to fit. It distorts some of it though. What would be a good non intrusive way to allow a user to open the full size version? Some type of hover? Would someone have some example code maybe? Thank you for any help.

**Comment by fnafox** [comment]
Make the scaled image link to the full image by nesting it.

```md
[![image.jpg](/assets/image.jpg =400x)](/assets/image.jpg)
```
